tracks/csharp/exercises/space-age/mentoring.md

```diff
@@ -0,0 +1,84 @@
+### Reasonable solution
+
+- using Dictionary
+    ```csharp
+    using System.Collections.Generic;
+
+    public class SpaceAge
+    {
+        private readonly double earthYear;
+
+        private static Dictionary<string, double> conversionFactor 
+                = new Dictionary<string, double>()
+        {
+            {"Mercury", 0.2408467},
+            {"Venus",   0.61519726},
+            {"Mars",    1.8808158},
+            {"Jupiter", 11.862615},
+            {"Saturn",  29.447498},
+            {"Uranus",  84.016846},
+            {"Neptune", 164.79132},
+        };
+
+        public SpaceAge(int seconds)
+        {
+            earthYear = seconds / 31557600.0;
+        }
+    
+        public double OnEarth() => earthYear;
+        public double OnMercury() => earthYear / conversionFactor["Mercury"];
+        public double OnVenus() => earthYear / conversionFactor["Venus"];
+        public double OnMars() => earthYear / conversionFactor["Mars"];
+        public double OnJupiter() => earthYear / conversionFactor["Jupiter"];
+        public double OnSaturn() => earthYear / conversionFactor["Saturn"];
+        public double OnUranus() => earthYear / conversionFactor["Uranus"];
+        public double OnNeptune() => earthYear / conversionFactor["Neptune"];
+    }
+    ```
+
+- or const members
+    ```csharp
+    using System;
+
+    public class SpaceAge
+    {
+        private double _seconds { get; }
+        private const int earthYearInSeconds = 31_557_600;
+        private const double orbitalMercury = 0.2408467;
```

Comment:
Maybe be a bit more explicit here: 

```suggestion
        private const double orbitalPeriodMercuryInEarthYears = 0.2408467;
```

This would also apply to the other constants.